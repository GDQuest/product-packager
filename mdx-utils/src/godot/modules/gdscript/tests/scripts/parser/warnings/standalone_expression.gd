func test():
	# The following statements should all be reported as standalone expressions:
	1234
	0.0 + 0.0
	Color(1, 1, 1)
	Vector3.ZERO
	[true, false]
	float(125)
