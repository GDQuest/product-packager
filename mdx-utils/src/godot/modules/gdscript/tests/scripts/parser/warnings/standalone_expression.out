GDTEST_OK
>> WARNING
>> Line: 3
>> STANDALONE_EXPRESSION
>> Standalone expression (the line has no effect).
>> WARNING
>> Line: 4
>> STANDALONE_EXPRESSION
>> Standalone expression (the line has no effect).
>> WARNING
>> Line: 6
>> STANDALONE_EXPRESSION
>> Standalone expression (the line has no effect).
>> WARNING
>> Line: 7
>> STANDALONE_EXPRESSION
>> Standalone expression (the line has no effect).
